layouts/shortcodes/changelog-table-row.html

```diff
@@ -13,13 +13,13 @@
 {{ end }}
 
 <tr>
-    <td><a href="https://github.com/pulumi/pulumi/blob/master/CHANGELOG.md#{{ $flatVersion }}-{{ $date }}">{{ $version }}</a></td>
+    <td><a id="cta-changelog-pulumi-sdk-version" href="https://github.com/pulumi/pulumi/blob/master/CHANGELOG.md#{{ $flatVersion }}-{{ $date }}" track-click>{{ $version }}</a></td>
     <td>{{ $date }}</td>
     <td>
-        <a href="https://get.pulumi.com/releases/sdk/pulumi-v{{ $version }}-linux-x64.tar.gz">Linux</a>
+        <a id="cta-changelog-pulumi-sdk-linux" href="https://get.pulumi.com/releases/sdk/pulumi-v{{ $version }}-linux-x64.tar.gz" track-click>Linux</a>
```

Comment:
So, I think we can definitely be in a place where we don't need _both_ a handcrafted `id` _and_ a `track-link` attribute. I think if we can define some sort of a schema that makes sense (and maybe this is better handled a `data-*` kind of thing), we can do one or the other, and have our script understand how to interpret that.

Reply:
Yup that is a much cleaner idea. Something like `data-tracking-id` would make sense to me. I've tried to maintain a schema of `${type}-${page}-${section}-${uniqueIdentifier}` with `uniqueIdentifier` being something to differentiate the element. Such as the button text or what the link does.

These short codes are a bit different because they're shared elements. So it is probably best to set the page value of the "id" in the javascript.

I am having a hard time trying to figure out a way where we don't have add some sort of attribute to all of our links. Maybe one of use will have an epiphany over the weekend :)